tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt5/MQTT5Test.java

```diff
@@ -871,4 +875,53 @@ public void testFlowControlAfterDisconnect() throws Exception {
       Wait.assertEquals(1L, () -> subscriptionQueue.getMessagesAcknowledged(), 2000, 50);
       Wait.assertEquals(0L, () -> subscriptionQueue.getMessageCount(), 2000, 50);
    }
+
+   /**
+    * This test helps validate that there are no longer timeouts related to persisting subscriptions under load.
+    * However, it is disabled because it takes a relatively long time to run. I'm leaving it here for ad hoc runs
+    * as necessary.
+    */
+   @Disabled
+   @Test
+   @Timeout(DEFAULT_TIMEOUT_SEC)
+   public void testLoadOnSubscriptionPersistence() throws Exception {
```

Comment:
Can you make this test faster?

I have a few SoakTests that will run faster than this one. it's taking 22 seconds..

You could perhaps limit the executor on a server and run less threads?


At least make it into a SoakTest

Reply:
700 milliseconds.. reproduced..  no need to make any other changes